packages/components/src/button/style.scss

```diff
@@ -119,13 +119,8 @@
 			background-size: 100px 100%;
 			// Disable reason: This function call looks nicer when each argument is on its own line.
 			/* stylelint-disable */
-			background-image: linear-gradient(
-				-45deg,
-				theme(button) 28%,
-				color(theme(button) shade(20%)) 28%,
-				color(theme(button) shade(20%)) 72%,
-				theme(button) 72%
-			);
+			background-image:
+ linear-gradient(-45deg, theme(button) 28%, color(theme(button) shade(20%)) 28%, color(theme(button) shade(20%)) 72%, theme(button) 72%);
```

Comment:
If we are going to reorganize the background-image in only one line then we should remove the comment which disables the style lint rule: